ClientLib/src/main/java/com/o3dr/services/android/lib/drone/companion/solo/tlv/SoloPanoStatus.java

```diff
@@ -0,0 +1,70 @@
+package com.o3dr.services.android.lib.drone.companion.solo.tlv;
+
+import android.os.Parcel;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Created by phu on 7/17/16.
+ */
+public class SoloPanoStatus extends TLVPacket {
+
+    private int currentStep;
+    public static final int MESSAGE_LENGTH = 2;
+
+    public int getCurrentStep() {
+        return currentStep;
+    }
+
+    private int totalSteps;
```

Comment:
@phmagic same comment as above, this variable should just be a byte since that's all you're storing.